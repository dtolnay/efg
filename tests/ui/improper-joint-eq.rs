use efg::efg;

#[efg(feature => "std")]
struct S;

fn main() {}
