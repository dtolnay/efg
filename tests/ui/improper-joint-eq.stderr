error: expected `=`
 --> tests/ui/improper-joint-eq.rs:3:15
  |
3 | #[efg(feature => "std")]
  |               ^^
